packages/manager/modules/iam/src/routes/policies.route.js

```diff
@@ -21,6 +22,13 @@ const state = () => ({
     ...asResolve(policiesComponent.resolves),
     ...asResolve(resolves),
   },
+  redirectTo: (transition) =>
+    transition
+      .injector()
+      .getAsync(`${hasPoliciesResolve.key}`)
+      .then((hasPolicies) =>
+        hasPolicies ? { state: ROUTES.POLICIES } : false,
```

Comment:
```suggestion
        !hasPolicies ? { state: ROUTES.ONBOARDING } : false,
```
Are you sure it shouldn't redirect to onboarding ?